projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/Project.java

```diff
@@ -98,8 +123,10 @@ protected void setProjectIntent(ProjectIntent projectIntent) {
    * @param projectIntent the intent of the project
    * @return a new project instance
    */
-  public static Project create(ProjectIntent projectIntent, ProjectCode projectCode) {
-    return new Project(ProjectId.create(), projectIntent, projectCode);
+  public static Project create(ProjectIntent projectIntent, ProjectCode projectCode,
```

Comment:
Why did you decide to go for a `PersonReference` instead of a `ProjectManager` and `PrincipalInvestigator` value object?

Reply:
it was purely technically driven. makes technical representation easier.